packages/client/tests/functional/queryRaw-send-type-hints/tests.ts

```diff
@@ -0,0 +1,74 @@
+import { setupTestSuiteMatrix } from '../_utils/setupTestSuiteMatrix'
+
+// @ts-ignore
+declare let prisma: import('@prisma/client').PrismaClient
+// @ts-ignore
+declare let Prisma: typeof import('@prisma/client').Prisma
+
+setupTestSuiteMatrix((suiteConfig) => {
+  test('Buffer ($queryRaw)', async () => {
+    if (suiteConfig['provider'] === 'mysql') {
+      await prisma.$queryRaw`INSERT INTO \`Entry\` (\`id\`, \`binary\`) VALUES (1, ${Buffer.from('hello')})`
```

Comment:
**question**: Are you specifically testing the escaped quotes here? I noticed that this one is using: 

```
(\`id\`, \`binary\`)
```

vs the one below: 

```
("id", "binary")
```

Reply:
MySQL and PostgreSQL have different syntax, so I have to choose between backticks and double quotes depending on the provider.